src/classes/BDI_FieldMappingCustomMetadata.cls

```diff
@@ -37,6 +37,27 @@
 
 public with sharing class BDI_FieldMappingCustomMetadata implements BDI_FieldMapping {
 
+    public static final String DEFAULT_DATA_IMPORT_FIELD_MAPPING_SET_NAME = 'Default_Field_Mapping_Set';
+
+    private static BDI_FieldMappingCustomMetadata fieldMappingInstance = null;
+    /*******************************************************************************************************
+    * @description flag to allow us to test out field level security exception
+    *
+    * TODO: refactor tests to make this unneccessary
+    */
+    public Boolean failFieldLevelSecurity = false;
```

Comment:
I think we can ALSO pull this out for this class as long as we don't want to use it for tests